ArkShop/ArkShop/Private/Store.cpp

```diff
@@ -16,8 +16,8 @@ namespace ArkShop::Store
 		if (!NoglinBuffClass)
 		{
 			try
-			{
-				FString buffClassString = "Blueprint'/Game/Genesis2/Dinos/BrainSlug/Buff_BrainSlugPostProccess.Buff_BrainSlugPostProccess'";
+			{				            
+				FString buffClassString = "Blueprint'/Game/Genesis2/Dinos/BrainSlug/Buff_BrainSlug_HumanControl.Buff_BrainSlug_HumanControl'";
```

Comment:
Are you positive this is the correct buff and not the original one used?

Reply:
Yes, the original Buff is for the player controlling the Noglin moving around, the new buff is for the player who is under Mind Control from the Noglin.

Reply:
I wonder if we need to check for both of these then to account for multiple situations.

Reply:
I think the main concern is the player being under mind control and then buying ascension kit from the shop or something like that to get out of the mind control. I'm not aware of any threats the other way around, but it might be nice to have the option anyway.